content/posts/2022-09-26-bundle-option-in-crc.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Different bundle types option in crc"
+date: 2022-09-26T10:41:10+05:30
+---
+
+Until 2.9 release CRC bundle config option is only able to accept local bundle path
+which user needs to download before consuming it. It has been around 3 releases since
+podman and okd bundles are migrated to quay instead of mirror location and team was
+brainstorming around how user can now download those bundles before consuming if that's
+only option.
+
```

Comment:
maybe start with a little information about the `bundle` config option, something like:
```
CRC provides a config option called `bundle` to configure the _location_ of the bundle
```

then the next para.. (modified after fixing some grammatical error and some rewording)

> Until 2.9.0 release of CRC the `bundle` option could only accept a local filesystem path, which meant users needed to download the bundle in advance before consuming it. The move to a container registry for hosting the `okd` and `podman` bundles, introduces another step in the process as bundles are now generated as an `OCI` container image and the `*.crcbundle` file needs to be extracted from the image